File: WW/app/src/main/java/com/ersubhadip/helpers/Storage.java
package com.ersubhadip.helpers;

import android.content.Context;
import android.content.SharedPreferences;
import android.graphics.Bitmap;
import android.util.Base64;

import java.io.ByteArrayOutputStream;

public class Storage {
    private SharedPreferences prefs;
    private SharedPreferences.Editor edit;
    private Context context;

    public Storage(Context context) {
        this.context = context;
        prefs = context.getSharedPreferences("userInfo", Context.MODE_PRIVATE);
        edit = prefs.edit();
    }

    public boolean isNewUser() {
        return prefs.getBoolean("isNewUser", true);
    }

    public void setIsNewUser(boolean b) {
        edit.putBoolean("isNewUser", b);
        edit.apply();
    }

    public String getNumber() {
        return prefs.getString("isAuthenticated", "");
    }

    public void setNumber(String b) {
        edit.putString("isAuthenticated", b);
        edit.apply();
    }

    public String getLat() {
        return prefs.getString("lat", "");
    }

    public void setLat(String b) {
        edit.putString("lat", b);
        edit.apply();
    }

    public String getLong() {
        return prefs.getString("long", "");
    }

    public void setLong(String b) {
        edit.putString("long", b);
        edit.apply();
    }

}

File: WW/app/src/main/java/com/ersubhadip/helpers/Storage.kt
package com.ersubhadip.helpers

import android.content.Context
import android.content.SharedPreferences
import android.graphics.Bitmap
import android.util.Base64
import java.io.ByteArrayOutputStream

class Storage {
    private var prefs: SharedPreferences? = null
    private var edit: SharedPreferences.Editor? = null
    private var context: Context? = null

    fun Storage(context: Context) {
        this.context = context
        prefs = context.getSharedPreferences("userInfo", Context.MODE_PRIVATE)
        edit = prefs?.edit()
    }

    fun isNewUser(): Boolean {
        return prefs!!.getBoolean("isNewUser", true)
    }

    fun setIsNewUser(b: Boolean) {
        edit!!.putBoolean("isNewUser", b)
        edit!!.apply()
    }

    fun getEmergencyNumber(): Boolean {
        return prefs!!.getBoolean("number", false)
    }

    fun setEmergencyNumber(b: Boolean) {
        edit!!.putBoolean("number", b)
        edit!!.apply()
    }

}